plugin/path_role.go

```diff
@@ -107,9 +170,57 @@ is assumed to be all service accounts under the role's project.`,
 			HelpSynopsis:    pathListRolesHelpSyn,
 			HelpDescription: pathListRolesHelpDesc,
 		},
+
+		// Edit service accounts on an IAM role
+		{
+			Pattern: fmt.Sprintf("role/%s/service-accounts", framework.GenericNameRegex("name")),
+			Fields: map[string]*framework.FieldSchema{
+				"name": {
+					Type:        framework.TypeString,
+					Description: "Name of the role.",
+				},
+				"add": {
+					Type:        framework.TypeCommaStringSlice,
+					Description: "Service-account emails or IDs to add.",
+				},
+				"remove": {
+					Type:        framework.TypeCommaStringSlice,
+					Description: "Service-account emails or IDs to remove.",
+				},
+			},
+			Callbacks: map[logical.Operation]framework.OperationFunc{
+				logical.UpdateOperation: b.pathRoleEditIamServiceAccounts,
+			},
+			HelpSynopsis:    "Add or remove service accounts for an existing `iam` role",
+			HelpDescription: "Add or remove service accounts from the list bound to an existing `iam` role",
+		},
+
+		// Edit service accounts on an GCE role
```

Comment:
Comment should say "labels" not "service accounts"

Reply:
done